adapters/handlers/rest/handlers_backup.go

```diff
@@ -28,16 +28,44 @@ type backupHandlers struct {
 	metricRequestsTotal restApiRequestsTotal
 }
 
+// compressionFromCfg transforms model backup config to a backup compression config
+func compressionFromCfg(cfg *models.BackupConfig) ubak.Compression {
+	if cfg != nil {
+		return ubak.Compression{
+			CPUPercentage: int(cfg.CPUPercentage),
+			ChunkSize:     int(cfg.ChunkSize),
+			Level:         parseCompressionLevel(*cfg.CompressionLevel),
```

Comment:
Any possibility this pointer in CompressionLevel is nil ? Or is swagger ensuring us it is always set ?

Reply:
it has a default, so it should be fine, it's just because a pointer to string